python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -297,8 +297,9 @@ def getImpl(v):
 
 class ProxyDataDescriptor(object):
 
-    def __init__(self, name):
+    def __init__(self, name, check_readOnly=True):
```

Comment:
What does this new parameter do? What has changed such that is is needed? It should also be called `check_read_only` for consistency (and PEP8).